packages/components/addon/components/hds/app-footer/legal-links.hbs

```diff
@@ -0,0 +1,16 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Link @href={{this.urlForSupport}} class="hds-app-footer__link--support">Support</Hds::AppFooter::Link>
```

Comment:
see my previous comment about these classes used only for testing purposes (as DOM selectors)

Reply:
Ok I deleted the class names. Not sure how to test that the links exist when passed in now.

Reply:
Resolving this for now. Can address in separate comments focused around tests.